x-pack/plugins/security_solution/public/resolver/store/data/selectors.test.ts

```diff
@@ -316,18 +317,209 @@ describe('data state', () => {
         requires a pending request to be aborted: {\\"databaseDocumentID\\":\\"first databaseDocumentID\\",\\"indices\\":[],\\"filters\\":{}}"
       `);
     });
-    describe('when after initial load resolver is told to refresh', () => {
-      const databaseDocumentID = 'doc id';
-      const resolverComponentInstanceID = 'instance';
-      const originID = 'origin';
-      const firstChildID = 'first';
-      const secondChildID = 'second';
-      const { resolverTree } = mockTreeWithNoAncestorsAnd2Children({
-        originID,
-        firstChildID,
-        secondChildID,
+    describe('and when the old request was aborted', () => {
```

Comment:
This looks worse than it is, I moved out a portion of the tests so they weren't nested under a `describe` block and added some tests for the time range filters.